lib/Transforms/Scalar/RewriteStatepointsForGC.cpp

```diff
@@ -69,7 +69,7 @@ namespace {
 struct RewriteStatepointsForGC : public FunctionPass {
   static char ID; // Pass identification, replacement for typeid
 
-  RewriteStatepointsForGC() : FunctionPass(ID) {
+  RewriteStatepointsForGC() : FunctionPass(ID), MustTrackBasePointers(true) {
```

Comment:
Yes. Passes can also be initialized through the `PassInfo` (see the LLVM docs for [llvm::Pass::lookupPassInfo](http://llvm.org/docs/doxygen/html/classllvm_1_1Pass.html#a992ff669acca94459037cfb1f41cb9dc) and for [llvm::PassInfo::getNormalCtor](http://llvm.org/docs/doxygen/html/classllvm_1_1PassInfo.html#aabe36c22724215bc13244e93fa598de5).